installation/install.sh

```
@@ -665,6 +665,9 @@ set_tomcat_config() {
     tomcat_config_path=$ZSTACK_INSTALL_ROOT/apache-tomcat/conf
     cp $tomcat_config_path/server.xml $tomcat_config_path/server.xml.bak || true
 
+    # set global var `MN_PORT`
+    get_mn_port()
```

Shellcheck (0.10.0)

[error] Couldn't parse this function. Fix to allow more checks.

(SC1073)